backend/app/Http/Controllers/API/TripController.php

```diff
@@ -208,7 +202,7 @@ public function show(Request $request, ContentProvider $contentProvider)
                 'created_by' => $trip->created_by,
                 'created_at' => $trip->created_at->toCookieString(),
                 'updated_at' => $trip->updated_at->toCookieString(),
-                'venues' => $venuesJSON
+                'schedule' => $trip->generateSchedule($contentProvider, false, true),
```

Comment:
Does this not generate a new schedule instead of using the existing scedule?

Reply:
No, that's why it sends false as an argument